src/core/point_cloud.py

```diff
@@ -12,3 +13,22 @@ def __init__(
 
     def read(self) -> o3d.geometry.PointCloud:
         return self._getter(self._path_to_pcd)
+
+    @staticmethod
+    def voxel_down_sample(
```

Comment:
The method seems to be out of place here, but on the other hand voxel down sample looks like an attribute of the point cloud. Maybe if we also implement lazy, it will be better